example/20-componentconfig-gardener-admission-controller-ipv6.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: admissioncontroller.config.gardener.cloud/v1alpha1
+kind: AdmissionControllerConfiguration
+gardenClientConnection:
+  qps: 100
+  burst: 130
+logLevel: info
+logFormat: json
+server:
+  webhooks:
+    bindAddress: "::"
```

Comment:
Why not `"[::]"` here as well?

Reply:
somehow the webhook gets handled differently and this will throw the error `listen tcp: address [[::]]:2721: missing port in address` because it adds the braces on its own, not like the other servers

Reply:
We just concatenated host and port together. I updated the PR and use the `net` package which the controller-runtime also uses to put host and port together for the webhook server. Now they all use `::` as the bind address :)